ui/cypress/e2e/dashboardsView.test.ts

```diff
@@ -151,7 +178,7 @@ describe('Dashboard', () => {
 
           // selected value in cell context is 2nd value
           cy.window()
-            .pipe(getSelectedVariable('veo', 0))
+            .pipe(getSelectedVariable(dashboard.id, 0))
```

Comment:
the concept of `veo` doesn't really exist any more, instead timemachine is loaded with a contextID (which is just the dashboardID) and loads it's truth from that instead